internal/core/domain/utxo_repository.go

```diff
@@ -74,7 +74,7 @@ type UtxoRepository interface {
 	UnlockUtxos(ctx context.Context, utxoKeys []UtxoKey) (int, error)
 	// DeleteUtxosForAccount deletes every utxo associated to the given account
 	// from the repository.
-	DeleteUtxosForAccount(ctx context.Context, accountName string) error
+	DeleteUtxosForAccount(ctx context.Context, namespace string) error
```

Comment:
The changes to this file are not so relevant. Can we revert them except for this one? Let's align it with the existing ones:
```suggestion
	DeleteUtxosForAccount(ctx context.Context, account string) error
```